recipe/meta.yaml

```
@@ -6,39 +6,56 @@ package:
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/googlemaps-{{ version }}.tar.gz
-  sha256: 3055fcbb1aa262a9159b589b5e6af762b10e80634ae11c59495bd44867e47d88
+  # Use GH release archive for tests
+  url: https://github.com/googlemaps/google-maps-services-python/archive/refs/tags/v{{ version }}.tar.gz
+  sha256: cc07f739731ba4e0a4586543ad14470695addff8f30840748b9c3b89e393aa36
+  patches:
+    - 0001-remove-superfluous-build-dependency.patch
```

nitpick: we place our patches in the folder `recipe/patches` (you would need to create a folder and move the patch file there).

The path will be relative to the `recipe` folder and can be expressed here as:

```suggestion
    - patches/0001-remove-superfluous-build-dependency.patch
```

---

Thanks ! I moved the path to a dedicated folder :)